src/main/java/com/bigcommerce/BigcommerceSdk.java

```diff
@@ -80,25 +46,31 @@
 
 	static final int TOO_MANY_REQUESTS_STATUS_CODE = 429;
 
+	public static final String VARIANTS = "variants";
+	public static final String CUSTOM_FIELDS = "custom_fields";
+
 	private static final String CATALOG = "catalog";
+	private static final String CATEGORIES= "categories";
 	private static final String SUMMARY = "summary";
 	private static final String PRODUCTS = "products";
+	private static final String CUSTOM_FIELDS_PATH = "custom-fields";
 	private static final String BRANDS = "brands";
 	private static final String ORDERS = "orders";
 	private static final String CUSTOMERS = "customers";
 	private static final String LIMIT = "limit";
 	private static final String PAGE = "page";
 	private static final String INCLUDE = "include";
-	private static final String VARIANTS = "variants";
 	private static final String SHIPPINGADDRESSES = "shipping_addresses";
 	private static final String SHIPMENTS = "shipments";
 	private static final String MIN_DATE_CREATED = "min_date_created";
 	private static final String STORE = "store";
 	private static final String METAFIELDS = "metafields";
 	private static final String IMAGES = "images";
-	private static final int MAX_LIMIT = 250;
+	private static final String PARENT_ID = "parent_id";
+	public static final int MAX_LIMIT = 250;
```

Comment:
I see where you are coming from, but when we used the library we put pagination outside and wanted to rely on a number that is more tightly bound to bigcommerce than our application.

Reply:
I'm not sure I understand. Can you please describe your use case on code outside of this file needing to know the max limit?   If calling code is concerned about the default max size, can't they just use the public methods that allow them to specify the limit?

Reply:
I can try. We had in our project something like

```java
public List<Category> getAllCategories(BigcommerceSdk client, Integer parentId) {
        return paginate(
                (page) -> client.getCategories(parentId, page, BigcommerceSdk.MAX_LIMIT),
                START_PAGE
        );
    }
```

where `paginate()` was a function that then used the information from the response to, well ... paginate :-)

We felt, that our application should not decide what's the paging limit.

Reply:
I see what you mean @brachetti . All i was getting at was to let the sdk do the magic, so your app doesn't need that parameter at all. From the above, it doesn't look like it really needs it.

We do similar things in our apps that use the sdk. See the getProducts method in the sdk. This hides the constant so the calling function just has to pass the page.
```
public Products getProducts(final int page, String... includes) {
    return getProducts(page, MAX_LIMIT, includes);
}
```

If you did this, your app wouldn't need to know the max limit and we'd be able to keep it private.

Reply:
I see what you are trying to do *nod*

What do you think about including pagination in the SDK itself, so you rather end up with a `getAllProducts()` method? I feel this would improve usability a lot

Reply:
It's in #5 , I'll base the new branch off this one, since I need the `PaginatedModel<T>` interface